miniscope_io/stream_daq.py

```diff
@@ -24,23 +24,13 @@
 
     HAVE_OK = True
 except (ImportError, ModuleNotFoundError) as ok_error:
-    module_logger = init_logger('uart_daq')
+    module_logger = init_logger('streamDaq')
     module_logger.warning(
         "Could not import OpalKelly driver, unable to read from FPGA!")
 
 # Parsers for daq inputs
 daqParser = argparse.ArgumentParser("stream_image_capture")
-daqParser.add_argument("source", help='Input source; ["UART", "OK"]')
-daqParser.add_argument("--port", help="serial port: string")
-daqParser.add_argument("--baudrate", help="baudrate: int")
-
-# Parsers for update LED
-updateDeviceParser = argparse.ArgumentParser("updateDevice")
-updateDeviceParser.add_argument("port", help="serial port")
-updateDeviceParser.add_argument("baudrate", help="baudrate")
-updateDeviceParser.add_argument("module", help="module to update")
-updateDeviceParser.add_argument("value", help="LED value")
-
+daqParser.add_argument("-c", "--config", help='YAML config file path: string')
```

Comment:
I like this change, but can we move away from argparser defined in module scope altogether? With this and the new script entrypoint, can we start grouping just the CLI logic in eg. A `miniscope_io.cli` directory so that this module is just a normal python module not intended to be run as `__main__`, and the `cli ` module has all the stuff for running it from command line?

Reply:
let's save this for a future PR where we start using `click`